build/build_parallel/config.json

```diff
@@ -57,6 +57,9 @@
       "directory": "device/core"
     },
     {
+      "directory": "service"
```

Comment:
>"directory": "service" [](start = 6, length = 22)

I'm curious to know why this ordering is significant.  AFAIK, the algorithm should be order-independent.